### ISSUE TITLE
Don't capitalise titles

### ISSUE THREAD

**Issue by neongreen** [evanrelf/r-haskell-theme#8]
It leads to cases like this one: https://www.reddit.com/r/haskell/comments/5c9cud/announcing_the_req_http_client_library/ (note `hTTP`)

And this one: https://www.reddit.com/r/haskell/comments/5awkhy/question_can_you_get_finergrained_control_of/ (`partialTypeSignature`)

**Comment by evanrelf** [comment]
I'm having trouble reproducing this issue, or at least understanding it. 

Are you saying that the title is being styled in a way that affects the capitalization? Or when you're inputting the words into the text box when creating a post, it affects the capitalization? 

I opened the links you provided on my desktop with styling enabled and compared it to in a Reddit app on my phone, and the capitalization is the same. Also when I disable styling it remains the same. So it seems like the capitalization is actually what the person typed and isn't being affected by the styling.

**Comment by neongreen** [comment]
@evanrelf Ah, I should've been more clear. 

> So it seems like the capitalization is actually what the person typed

This is exactly what the problem is. The person enters `h`, sees `H`, enters `ttp` without holding Shift, sees `Http`, changes last three letters to `TTP`, presses “submit” and it gets submitted as `hTTP`. (Titles can't be edited after submission.)

**Comment by neongreen** [comment]
From the second link:

> I didn't realise that I hadn't camelcase'd `partial`

So, it's not about code messing up people's titles, it's about people messing up their own titles and not realising it because `text-transform: capitalize` in the input box hides it from them.

**Comment by evanrelf** [comment]
Yeah that makes sense, I understand now and am able to reproduce the issue. I'm on my phone at work right now but I'll get to it as soon as I can!
